pkg/gardenlet/operation/botanist/shootsystem_test.go

```diff
@@ -87,7 +67,7 @@ var _ = Describe("ShootSystem", func() {
 				},
 			}
 
-			shootSystem.EXPECT().SetAPIResourceList(apiResourceList)
+			shootSystem.EXPECT().SetAPIResourceList(nil)
```

Comment:
We could also use something like https://github.com/gardener/gardener/blob/ceb8906549ddce46a18e2ec2c9583f483d55998a/pkg/utils/gardener/secretsrotation/etcd_test.go#L377-L381